docs/database/ftl.md

```diff
@@ -130,6 +130,8 @@ ID | Status | | Details
 12 | Allowed | ✅ | Retried query
 13 | Allowed | ✅ | Retried but ignored query (this may happen during ongoing DNSSEC validation)
 14 | Allowed | ✅ | Already forwarded, not forwarding again
+15 | Blocked | ❌ | Blocked (database is busy)
```

Comment:
It may or not be blocked depending on whatever the user considers more appropriate. See https://docs.pi-hole.net/ftldns/configfile/#reply_when_busy

Reply:
I've been thinking about this too. However, status 15 does always mean it was blocked. If users change the setting, the query won't get that status....
But I'll add a not that this is configurable.